boot-qemu-help.txt

```diff
@@ -0,0 +1,42 @@
+Usage: ./boot-qemu.sh <options>
+
+Script description: Boots a Linux kernel in QEMU.
+
+Required parameters:
+  -a | --arch | --architecture:
+    The architecture to boot. Possible values are:
+       * arm32_v5
+       * arm32_v6
+       * arm32_v7
+       * arm64
+       * mips
+       * mipsel
+       * ppc32
+       * ppc64
+       * ppc64le
+       * x86_64
+
+  -k | --kbuild-folder:
+    The kernel build folder, as an absolute path or relative path
+    from wherever the script is being run. This is wherever the
+    compiled vmlinux image lives, not the architecture's boot folder.
+
+Optional parameters:
+  -d | --debug:
+    Invokes 'set -x' for debugging the script.
+
+  -h | --help:
+    Prints this message then exits.
+
+  -i | --interactive | --shell:
+    By default, the rootfs images in this repo just boots the kernel,
+    print the version string, then exit. If you would like to actually
+    interact with the machine, this option passes 'rdinit=/bin/sh' to
+    the kernel command line so that you are thrown into an interactive
+    shell.
+
+  -t | --timeout:
+    By default, the timeout command waits 3 minutes before killing the
+    QEMU machine. Depending on the power of the host machine, this might
+    not be long enough for a kernel to boot so this allows that timeout
+    to be configured. Takes the value passed to timeout (e.g. 30s or 4m).
```

Comment:
does `-i` disable the timeout?

Reply:
Yes: https://github.com/nathanchance/boot-utils/blob/101ef4f4258719dd4879cf68ae28af8cc9351595/boot-qemu.sh#L178

Will push a patch to clarify.